client/layout/guided-tours/tours/tutorial-site-preview-tour.js

```diff
@@ -41,10 +43,10 @@ export const TutorialSitePreviewTour = makeTour(
 		>
 			<p>
 				{ translate(
-					'{{strong}}View Site{{/strong}} shows you what your site looks like to visitors. Click it to continue.',
+					'{{viewSiteButton/}} shows you what your site looks like to visitors. Click it to continue.',
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( '{{ViewSiteButton/}} shows you what your site looks like to visitors. Click it to continue.' )` *ES Score: 17*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).